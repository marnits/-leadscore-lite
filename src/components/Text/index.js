import React from 'react';
import PropTypes from 'prop-types';
import styles from './styles.module.scss';

const Text = ({ text, strong }) => (
  <span className={`${styles.text} ${strong && styles.strong}`}>{text}</span>
);

Text.defaultProps = {
  strong: false,
};

Text.propTypes = {
  text: PropTypes.string.isRequired,
  strong: PropTypes.bool,
};

export default Text;
