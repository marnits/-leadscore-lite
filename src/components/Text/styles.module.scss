.text {
  color: #131313;
  font-size: 14px;
  line-height: 18px;
}

.strong {
  font-weight: 700;
}
