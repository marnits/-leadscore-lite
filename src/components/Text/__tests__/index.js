import React from 'react';
import renderer from 'react-test-renderer';
import Text from '../index';

describe('Text', () => {
  test('should render component properly', () => {
    const tree = renderer.create(<Text text="text" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
