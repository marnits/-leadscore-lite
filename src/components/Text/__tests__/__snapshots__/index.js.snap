// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text should render component properly 1`] = `
<span
  className="text false"
>
  text
</span>
`;
